input/pagecontent/modifiers.md

```diff
@@ -284,24 +200,49 @@
   style='font-size:9.0pt;mso-bidi-font-size:10.0pt;line-height:115%'><o:p>&nbsp;</o:p></span></p>
   </td>
  </tr>
- <tr style='mso-yfti-irow:11;mso-yfti-lastrow:yes'>
-  <td width="19%" colspan=2 style='width:19.22%;border-top:none;border-left:
-  solid #8EAADB 1.0pt;border-bottom:solid #8EAADB 1.0pt;border-right:none;
-  mso-border-top-alt:solid #8EAADB .5pt;mso-border-top-alt:solid #8EAADB .5pt;
-  mso-border-left-alt:solid #8EAADB .5pt;mso-border-bottom-alt:solid #8EAADB .5pt;
-  padding:0cm 5.4pt 0cm 5.4pt'>
+
+ <tr style='mso-yfti-irow:3'>
+  <td width="13%" style='width:13.86%;border-top:none;border-left:solid #8EAADB 1.0pt;
+  border-bottom:solid #8EAADB 1.0pt;border-right:none;mso-border-top-alt:solid #8EAADB .5pt;
+  mso-border-top-alt:solid #8EAADB .5pt;mso-border-left-alt:solid #8EAADB .5pt;
+  mso-border-bottom-alt:solid #8EAADB .5pt;padding:0cm 5.4pt 0cm 5.4pt'>
   <p class=MsoNormal align=left style='text-align:left;mso-yfti-cnfc:4'><span
   class=GramE><b><span style='font-size:9.0pt;mso-bidi-font-size:10.0pt;
-  line-height:115%'></span></b></span><b><span style='font-size:9.0pt;
-  mso-bidi-font-size:10.0pt;line-height:115%'><span class=SpellE></span><span
-  class=SpellE></span><span class=SpellE>of-type</span><o:p></o:p></span></b></p>
+  line-height:115%'>:<span class=SpellE>iterate</span></span></b></span><b><span
```

Comment:
pas de lien vers la spec. Est-ce que vous voulez mettre le lien vers R5 comme pour http://hl7.org/fhir/search.html#modifieroftype ?

Reply:
sauf si pour oftype ce n'était pas volontaire ?
<img width="554" alt="image" src="https://github.com/ansforge/IG-fhir-repertoire-offre-ressources-sante/assets/117643165/dd40925c-f9f2-4bb4-9dad-a490581112be">
et dans  ce cas on met aussi above ?

Reply:
@sdemeyANS on pourrait mettre le lien général de la documentation de search en R4 (https://www.hl7.org/fhir/R4/search.html) pour iterate, of-type et above ?

Reply:
si pour oftype ce n'était pas volontaire et si on veut être homogène avec les autre liens oui. @paulineheurtebise et @nriss pour avis :)